src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx

```diff
@@ -23,6 +23,23 @@ export default class Table extends React.Component {
     location: PropTypes.object,
   };
 
+  state = {
+    isChangingTabs: false,
+  };
+
+  componentDidUpdate(prevProps) {
+    const tabChanged = prevProps.view.id !== this.props.view.id;
+    if (tabChanged) {
```

Comment:
There's a scenario that `this.state.isChangingTabs = true` and `tabChanged = true`, thereby leading to an extra re-render thru the `setState()` call.

```suggestion
    if (!this.state.isChangingTabs && tabChanged) {
```

Reply:
Ty!